### PR TITLE
analytic enhancement

### PR DESCRIPTION
**Pull request by luc-demeyer**
addition of analytic account on the asset. 
This field is 'defaulted' from the asset category.

### CODE REVIEW

File: account_asset_management/account_asset_view.xml [hunk 1]
```diff
@@ -126,8 +126,9 @@
                 </group>
                 <separator string="Other Information" colspan="4"/>
                 <group colspan="4" col="4">
-                  <field name="category_id" on_change="onchange_category_id(category_id)" attrs="{'required':[('type','=','normal')]}" colspan="4"/>
+                  <field name="category_id" on_change="onchange_category_id(category_id)" attrs="{'required':[('type','=','normal')]}"/>
                   <field name="partner_id"/>
+                  <field name="account_analytic_id" groups="analytic.group_analytic_accounting"/>
```
Comment (adrienpeiffer) [inline account_asset_management/account_asset_view.xml:131]: Can you add a domain to avoid to select view account ?

**Comment by adrienpeiffer** [comment]
:+1: (Code review and functionnal test)

File: account_asset_management/account_asset_view.xml (hunk 1 above)
Comment (adrienpeiffer) [inline account_asset_management/account_asset_view.xml:131]: It's done in python. Thanks !

**Comment by chrisdec** [comment]
:+1: (functional test)

**Comment by luc-demeyer** [comment]
It looks like this one is  ready for merge.
Can someone do this ?

**Comment by pedrobaeza** [comment]
Travis is not going to complain about this, so I merge.

**Pull request merged**